src/__tests__/build-parameters.test.ts

```diff
@@ -0,0 +1,62 @@
+import rimrafCB from 'rimraf';
+import {exec as execCB, ExecOptions} from 'child_process';
+import * as path from 'path';
+import {promisify} from 'util';
+
+const rimraf = promisify(rimrafCB);
+
+const testDir = path.join(__dirname, 'build-parameters');
+
+const exec = (cmd: string, options: ExecOptions): Promise<string> =>
+  new Promise((resolve, reject) => {
+    execCB(cmd, options, (error, stdout, stderr) => {
+      if (error) {
+        reject(stderr || stdout || error.message);
+      } else {
+        resolve(stdout);
+      }
+    });
+  });
+
+const buildProject = async (project: string) => {
+  await exec(`cp tsconfig.${project}.json tsconfig.json`, {cwd: testDir});
```

Comment:
One problem I noticed is that `typescript-json-validator` will pull from the project's `tsconfig.json` file, even if the project was run with a different config file, IE, `tsconfig.test.json`.